### PR TITLE
Added Hindi language support

### PR DESCRIPTION
**Pull request by SourabhRajput123**
- Added Hindi (hi) translations to the project
- Updated supportedLanguages map to include Hindi
- Integrated Hindi translations in the resources object for i18n

### CODE REVIEW

**Comment by imblowfish** [comment]
Hello @SourabhRajput123
Thank you for your contribution
Can you please tel why did you closed this PR? Everything looks great.

Is anything wrong on our side? Maybe something is not set according to Hacktoberfest rules?

**Comment by SourabhRajput123** [comment]
Hello, @imblowfish 

Thank you for your message. I apologize for closing the PR; it was due to some confusion on my part. I appreciate your understanding and the opportunity to contribute to the project.

If everything looks good, I’d be happy to reopen the PR. Please let me know if there’s anything specific you’d like to discuss!

Best regards,
Sourabh Rajput

**Comment by IraSoro** [comment]
Hello, @SourabhRajput123! Thanks for contributing to the project!

It looks like your PR didn't pass the auto checks.  This is our first time participating in Hacktoberfest. We have rewritten the `CONTRIBUTING.md` file with more precise instructions.

If it's not difficult, please fix what is described in sections 8, 10, 11 in [CONTRIBUTING.md](https://github.com/IraSoro/peri/blob/master/CONTRIBUTING.md). Or I can do it myself.

**Comment by SourabhRajput123** [comment]
Hello, @IraSoro 

Thank you for the feedback! I appreciate the input. This is my first time participating in Hacktoberfest as well, and I’m new to open source. I’ll review the updated CONTRIBUTING.md file and do my best to address the issues in sections 8, 10, and 11.

Thanks again for your guidance!

Best regards,
Sourabh Rajput

File: package.json [hunk 1]
```diff
@@ -81,6 +81,7 @@
     ]
   },
   "devDependencies": {
+    "@babel/plugin-proposal-private-property-in-object": "^7.21.11",
```
Comment (imblowfish) [inline package.json:84]: Why do you need this package here?

**Review by imblowfish — CHANGES_REQUESTED** [review]
Could you please remove unused packages in package.json file

File: package.json (hunk 1 above)
Comment (SourabhRajput123) [reply package.json:84]: I sincerely apologize if I overstepped. I was attempting to troubleshoot a bug that I believed was caused by my local environment. In hindsight, I realize that this package is not necessary for your application, and I apologize once again for any inconvenience caused and for my unintentional unprofessional behavior.

Comment (IraSoro) [reply package.json]: @SourabhRajput123, everything looks good! Thanks for fixing that. 
Could you also please remove the changes from the `package-lock.json` file?

File: package-lock.json [hunk 2]
```diff
@@ -61,6 +61,7 @@
         "workbox-streams": "^7.0.0"
       },
       "devDependencies": {
+        "@babel/plugin-proposal-private-property-in-object": "^7.21.11",
```
Comment (IraSoro) [inline package-lock.json:64]: Please remove all changes to this file.

**Review by IraSoro — CHANGES_REQUESTED** [review]

**Comment by imblowfish** [comment]
Hello @SourabhRajput123! If you have any problems with `package-lock.json` changes reverting we can do this ourselves

**Comment by SourabhRajput123** [comment]
Hello! @imblowfish 
Thank you for your offer to help. If you’d like, you can go ahead and revert the changes to package-lock.json, as I’m having difficulty doing it myself. I appreciate your assistance!

**Comment by imblowfish** [comment]
@IraSoro could you please update `package-lock.json`? I don't have access to my personal laptop

**Comment by IraSoro** [comment]
@imblowfish, check please.

**Review by IraSoro — APPROVED** [review]

**Review by imblowfish — APPROVED** [review]

**Comment by IraSoro** [comment]
@SourabhRajput123, thank you and congratulations on your first contribution to the `Peri` project!

**Pull request merged**